pages/management/index.vue

```diff
@@ -130,7 +131,8 @@ export default Vue.extend({
         {
           hid: 'description',
           name: 'description',
-          content: 'Login with your favorite wallet provider to manage your project or proposal.',
+          content:
+            'Login with your favorite wallet provider to manage your project or proposal.',
```

Comment:
nitpick(blocking): we use the same string 3 times, in cases like this we should define a const that we can reference. Also can we use the  an i18n variable for this?

Reply:
Created an new ticket to align this for the entire project: #458